runtime/testy/runtimeUtil.py

```diff
@@ -59,7 +59,7 @@ def __new__(cls):
   GET_TIME            = ()
 
 class RUNTIME_CONFIG(Enum):
-  STUDENT_CODE_TIMEOUT        = 3
+  STUDENT_CODE_TIMEOUT        = 1
```

Comment:
BTW the name `STUDENT_CODE_TIMEOUT` is too similar to `BAD_EVENTS.STUDENT_CODE_TIMEOUT`. Also, why is `RUNTIME_CONFIG` an Enum? That makes no sense. I strongly recommend cleaning this up in a future PR.